index.html

```diff
@@ -3879,8 +3940,7 @@ <h2 id="GainNode">
           </dt>
           <dd>
             <p>
-              Represents the amount of gain to apply. Its default
-              <code>value</code> is 1 (no gain change). This parameter is
```

Comment:
Yeah, I didn't like specifying the defaults in two places, but what can we do now?